src/server/qgsserverplugins.h

```diff
@@ -40,7 +40,7 @@ class SERVER_EXPORT QgsServerPlugins
     //! Pointer to QgsPythonUtils
     static QgsPythonUtils* mPythonUtils;
     //! List of available server plugin names
-    static QStringList mServerPlugins;
```

Comment:
Same here - I'd suggest still returning the copy, as again the qt containers are implicitly shared